src/constants/ui.ts

```diff
@@ -1,2 +1,3 @@
 export const MIN_BUTTON_SIZE = 48
+export const TABLET_WIDTH_THRESHOLD = 767
```

Comment:
I think it is better to make another PR for Tablet UI supports

Reply:
> I think it is better to make another PR for Tablet UI supports

Ok I will create another PR for it:)